website/versioned_docs/version-0.77/getting-started.md

```diff
@@ -26,7 +39,7 @@ Remember to call `@react-native-community/cli init` from the place you want your
 <!-- See https://www.npmjs.com/package/react-native?activeTab=versions for the RN version tags. -->
 
 ```bat
-npx --yes @react-native-community/cli@next init <projectName> --version "latest"
+npx --yes @react-native-community/cli@next init <projectName> --version "0.77.1"
```

Comment:
This should be "0.77-stable" but Meta hasn't made that tag yet, just pinged them to create it

Reply:
If you are manually creating a 0.77-stable app here, I'm not sure if you need to add the warning above, the app will always be 0.77-stable (and should be since it's the 0.77 documentation) and not use latest.